src/client/eppo-client.ts

```diff
@@ -988,6 +1000,32 @@ export default class EppoClient {
     validateNotBlank(flagKey, 'Invalid argument: flagKey cannot be blank');
 
     const flagEvaluationDetailsBuilder = this.newFlagEvaluationDetailsBuilder(flagKey);
+    const overrideVariation = this.overridesStore?.get(flagKey);
+    if (overrideVariation) {
+      const overrideAllocationKey = 'override-' + overrideVariation.key;
+      const flagEvaluationDetails = flagEvaluationDetailsBuilder
+        .setMatch(
+          0,
+          overrideVariation,
+          { key: overrideAllocationKey, splits: [], doLog: false },
+          null,
+          undefined,
+        )
+        .build('MATCH', 'Flag override applied');
```

Comment:
To simplify testing: consider a new method on `FlagEvaluationDetailsBuilder` for returning an over-ride match.